### ISSUE TITLE
[3][kapeer42] Proxy Change Issue

### ISSUE THREAD

**Issue by startailcoon** [bitshares/bitshares-ui#1799]
There is an issue when changing proxy for voting. Setting proxy works fine, but the app fails when trying to input a new proxy account.

It's also difficult to remove the proxy all together, removing the input and save isn't an obvious way to do it. We should add a button for nullifying the proxy selection.

## Todo
- [ ] Enable easy removal of proxy, setting no proxy
- [ ] Fix changing of proxy

**Comment by startailcoon** [comment]
It's yours @kapeer42

**Comment by startailcoon** [comment]
> Are you talking about that?

Yes. In my case it was when I tried to change the proxy to another name by highlighting the current proxy name. When I started typing, one of the hooks got stuck in an endless loop.

Setting proxy for the first time didn't give me an issue, but I haven't tested it enough to say it doesn't have issues.

**Comment by kapeer42** [comment]
Ok I'm able to reproduce it now, it only happens on main net for me and boii it's nasty.

First I reproduced it in Firefox then I hoped I could get a better stack trace with chrome but chrome did nothing to prevent the page from sucking up all resources and it ended up crashing my computer with it.

I found a fix, will push soon.

By the way there is a bug in the `AccountSelector` component when the `_accountName` is an empty string, I was wondering if you were talking about that at first but no.

Also the blank page bug I got earlier is something else entirely too.
